app/views/smart_answers/shared/_previous_answers.html.erb

```diff
@@ -40,10 +40,17 @@
       edit: {
         href: @presenter.change_answer_link(question, response_store.forwarding_responses),
         data_attributes: {
-          "module": "gem-track-click",
+          "module": "gem-track-click ga4-link-tracker",
           "track-category": "Smart Answer Change Link",
           "track-action":  "Link clicked",
-          "track-label": question.change_link_track_label(accepted_response)
+          "track-label": question.change_link_track_label(accepted_response),
+          ga4_link: {
+            event_name: "form_change_response",
+            type: "smart answer",
+            section: question.change_link_track_label(accepted_response),
+            action: "change response",
+            tool_name: @presenter.title,
```

Comment:
I think we might need to set the `text` property here to override the link text. I've checked the implementation guide and the `text` property is set as the answer rather than the question.

Reply:
Good spot, thanks. Just checked and its not hard to implement, but it raises a potential PII issue, I'll check with Harriet.

Reply:
Have agreed to not include the question answer in this tracking as it could contain PII and we have no way (at this point in the process) to determine that.

Reply:
@JamesCGDS are you happy to approve?